src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,32 @@
+import { useContext } from 'react';
+import { TodoItem } from '../TodoItem';
+import { TodosContext } from '../TodoContext/TodoContext';
+import { Status } from '../../types/Status';
+
+export const TodoList: React.FC = () => {
+  const { todos, filterStatus, tempTodo } = useContext(TodosContext);
+
+  const prepearedTodos = filterStatus === Status.All
+    ? todos
+    : todos.filter(todo => {
+      switch (filterStatus) {
+        case Status.Active:
+          return !todo.completed;
+
+        case Status.Completed:
+          return todo.completed;
+
+        default:
+          return true;
+      }
+    });
+
+  return (
+    <section className="todoapp__main" data-cy="TodoList">
```

Comment:
Use semantic tags for list `ul` `li`. Don't forget to remove default `li` dots and `ul` paddings